docs/getting-started/install/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Install Shopify App CLI
+section: getting-started
+toc: false
+redirect_from: "/install/"
+---
+
+Shopify App CLI can be installed using a variety of package managers.
+
+### Brew (MacOS)
```

Comment:
Extreme nitpick: I think officially it is macOS (lowercase 'm')

Reply:
oooo good catch